gradle.properties

```diff
@@ -21,6 +21,7 @@ group=org.apache.kafka
 #  - tests/kafkatest/version.py (variable DEV_VERSION)
 #  - kafka-merge-pr.py
 version=2.4.0-SNAPSHOT
+latestRelease=2.2.0
```

Comment:
Should we call this `latestReleaseVersion`?

Reply:
ok